### PR TITLE
Ltmbr gletschermaechtigkeit BGDIDIC-1927

### PR DESCRIPTION
**Pull request by marionb**
test link: https://mf-chsdi3.dev.bgdi.ch/ltmbr_gletschermaechtigkeit_BGDIDIC-1927/shorten/8d2761f5be

### CODE REVIEW

**Review by b4l — APPROVED** [review]
Floating point attributes are generally nicer rounded in the tooltip. Otherwise it looks good!

**Review by ltclm — CHANGES_REQUESTED** [review]
nice work thank you!
concerning technical implementation i have nothing to add.

The only concern i have is the **precision of the attributes**.
Is it really intended to show that precise values? These values have to be rounded i think. But you have to ask the Data owner about that. if not, im really impressed that we're able to measure thinkness with nanometer precision, respect!

**Comment by marionb** [comment]
@ltclm  The precision is rounding to the nearest whole number. Although there is still a trailing zero in the tool-tip. I could not get rid of it when trying to import the numbers as `Integers` in the model so I am guessing I will have to change the data type in the DB. Not sure if it is really worth it so I left it for the moment.

**Review by faselm — APPROVED** [review]

File: chsdi/templates/htmlpopup/gletschermaechtigkeit_gpr_profiles.mako [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c,lang)">
+<%
+  layer = 'ch.swisstopo.geologie-gletschermaechtigkeit.'
```
Comment (ltclm) [inline chsdi/templates/htmlpopup/gletschermaechtigkeit_gpr_profiles.mako:5]: you can use `c['layerBodId']` for the layername

File: chsdi/templates/htmlpopup/gletschermaechtigkeit_ice_thikness.mako [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c,lang)">
+<%
+  layer = 'ch.swisstopo.geologie-gletschermaechtigkeit.'
```
Comment (ltclm) [inline chsdi/templates/htmlpopup/gletschermaechtigkeit_ice_thikness.mako:5]: you can use `c['layerBodId']` for the layername

**Review by ltclm — CHANGES_REQUESTED** [review]

File: chsdi/templates/htmlpopup/gletschermaechtigkeit_ice_thikness.mako [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 <%def name="table_body(c,lang)">
 <%
-  layer = 'ch.swisstopo.geologie-gletschermaechtigkeit.'
+  layer = c['layerBodId'] + '.'
 %>
 <tr><td class="cell-left">${_(layer + 'pk_sgi')}</td> <td>${c['attributes']['pk_sgi'] or '-'}</td></tr>
```
Comment (ltclm) [inline chsdi/templates/htmlpopup/gletschermaechtigkeit_ice_thikness.mako:7]: here you should add the dot

**Review by ltclm — CHANGES_REQUESTED** [review]

**Review by ltclm — APPROVED** [review]

**Comment by ltclm** [comment]
@marionb could you please try to resolve the conflicts

**Comment by rebert** [comment]
rebased and merged here: Ltrea merge gletscher #3592

**Pull request closed without merging**